### PR TITLE
OutputWriters for MultiRegion

### PR DESCRIPTION
**Pull request by simone-silvestri**
this PR fixes a bug in the `OutputWriters` for which the output fields are not updated when writing

### CODE REVIEW

File: src/MultiRegion/multi_region_output_writers.jl [hunk 1]
```diff
@@ -6,6 +6,8 @@ import Oceananigans.OutputWriters:
                         serializeproperty!
 
 # This is working just fine at the moment?
+# But it will be veeeeery slow, as reconstruct_global_field is not 
+# a performant operation
```
Comment (glwagner) [inline src/MultiRegion/multi_region_output_writers.jl:10]: Good!

File: src/MultiRegion/multi_region_output_writers.jl [hunk 2]
```diff
@@ -6,6 +6,8 @@ import Oceananigans.OutputWriters:
...
 function fetch_output(mrf::MultiRegionField, model)
     field = reconstruct_global_field(mrf)
     compute_at!(field, time(model))
```
Comment (glwagner) [inline src/MultiRegion/multi_region_output_writers.jl:13]: Another possibility is to `@apply_regionally` to `fetch_output` right?

Then maybe we also want to extend `convert_output` to handle a `MultiRegionObject` or something (eg just save as a tuple, converting each element).

**Comment by glwagner** [comment]
You can also considering transferring to the CPU first, and then carrying on with GPU calculations while the CPU does `reconstruct_global_field` and then saves to disk.

**Comment by simone-silvestri** [comment]
Let s merge this to have at least an output for multi region (I would need it to debug the multi gpu global simulation).

Then we can design a new method that works in the context of cubed spheres also

**Review by glwagner — APPROVED** [review]

**Pull request merged**